h2o-extensions/target-encoder/src/main/java/hex/schemas/TargetEncoderV3.java

```diff
@@ -33,21 +26,13 @@
     public String[] fields() {
       final List<String> params = extractDeclaredApiParameters(getClass());
       params.add("model_id");
+      params.add("ignored_columns");
```

Comment:
I don't think this one needs to be added here: for other algos, it is "write-only".
Adding it as a field mainly causes its exposition as a public property in Python, why would you need this?